boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059.rst

```diff
@@ -45,16 +45,37 @@ hardware features:
 +-----------+------------+----------------------+
 | Interface | Controller | Driver/Component     |
 +===========+============+======================+
-| NVIC      | on-chip    | nested vectored      |
-|           |            | interrupt controller |
 +-----------+------------+----------------------+
-| RTC       | on-chip    | system clock         |
+| ADC       | on-chip    | adc                  |
 +-----------+------------+----------------------+
-| GPIO      | on-chip    | gpio                 |
+| CLOCK     | on-chip    | clock_control        |
 +-----------+------------+----------------------+
 | FLASH     | on-chip    | flash                |
 +-----------+------------+----------------------+
-| RADIO     | on-chip    | Bluetooth            |
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
```

Comment:
This board doesn't have a Segger chip.

Reply:
oops! well spotted, thank you will fix now.